log_extractor.py

```diff
@@ -436,8 +449,52 @@ def parse_logs(self):
                             )
 
 
-# helper.download_artifact(job_url=TEST_URL, dst=TEST_DST)
-log_extractor = LogExtractor(TEST_DST, logs=TEST_LOGS)
-# log-extractor.extract_all(path=TEST_DST)
-log_extractor.parse_art_logs()
-log_extractor.parse_logs()
+@click.command()
+@click.option("--job", help="Job name", required=True)
+@click.option(
+    "--build", help="build number of the job", required=True, type=int
+)
+@click.option(
+    "--folder", help="Folder path to save the logs",
+    default=os.path.join(user.home, "art-tests-logs")
+)
+def run(job, build, folder):
+    """
+    Extract logs from Jenkins jobs.
+    """
+    def get_jenkins_connection():
+        """
+        Get Jenkins connection object
+
+        Returns:
+            Jenkins: Jenkins connection
+        """
+        ssl._create_default_https_context = ssl._create_unverified_context
+        server = "https://rhev-jenkins.rhev-ci-vms.eng.rdu2.redhat.com"
+        return jenkins_api.Jenkins(url=server)
+
+    jenkins_connection = get_jenkins_connection()
+    build_info = jenkins_connection.get_build_info(name=job, number=build)
+    job_url = build_info.get("url")
+
+    if not os.path.exists(path=folder):
+        os.mkdir(folder)
+
+    job_folder = os.path.join(folder, job)
+    if not os.path.exists(path=job_folder):
+        os.mkdir(job_folder)
+
+    build_folder = os.path.join(job_folder, str(build))
+    if not os.path.exists(path=build_folder):
+        os.mkdir(build_folder)
+
+    helper.download_artifact(job_url=job_url, dst=build_folder)
+    log_extractor = LogExtractor(dst=build_folder, logs=TEST_LOGS)
```

Comment:
I wanted to send it as part of cli arguments --logs=vdsm.log,engine.log,...
What do you think?